### DISCUSSION TITLE
I got stuck in a solar post

### DISCUSSION THREAD

**Discussion by stefanesa** [polymathrobotics/caladan_examples#12] [Q&A] [answered]
I got stuck in the solar post

![Screen Shot 2022-07-22 at 12 13 03 PM](https://user-images.githubusercontent.com/1851242/180508681-31cc8126-db86-4a7c-98f2-b39be20ee058.png)

**Comment by iliabaranov** [comment, marked as answer]
I've noticed that, if the simulation is left alone for a long (IE hours) time, the vehicle will drift out of position (grrr Gazebo physics :P).
This drift in turn eventually confuses the costmap/path planner, since it's beyond reasonable distance.

My recommendation is to reset the world if the simulation has been left alone for a while, and then start to test with it.
